src/workerd/server/bundle-fs-test.c++

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2024-2029 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include <workerd/server/bundle-fs.h>
```

Comment:
I think all fs related files should have a prefix of `fs-` or it will be extremely hard to navigate through multiple fs related implementations.

```suggestion
#include <workerd/server/fs-bundle.h>
```